src/layer/tile/GridLayer.js

```diff
@@ -311,7 +315,8 @@ L.GridLayer = L.Layer.extend({
 	_initContainer: function () {
 		if (this._container) { return; }
 
-		this._container = L.DomUtil.create('div', 'leaflet-layer');
+		this._container = L.DomUtil.create('div', 'leaflet-layer ' + this.options.className);
```

Comment:
This might evaluate to the string `"leaflet-layer undefined"` or `"leaflet-layer false"` in some edge cases. See https://github.com/Leaflet/Leaflet/blob/master/src/layer/Popup.js#L288 and https://github.com/Leaflet/Leaflet/blob/master/src/layer/marker/Icon.js#L108